client/src/utils/mutations.js

```diff
@@ -0,0 +1,46 @@
+import { gql } from '@apollo/client';
+
+export const ADD_PROFILE = gql`
+  mutation addProfile($username: String!, $email: String!, $password: String!) {
+    addProfile(name: $username, email: $email, password: $password) {
+      token
+      profile {
+        _id
+        name
+      }
+    }
+  }
+`;
```

Comment:
The backend is doing firstName and lastName instead of username, let's discuss which one we would like to go with.

Reply:
Let's stay with what you guys have going on the backend.